pysquared/config/radio.py

```diff
@@ -17,6 +18,7 @@ def __init__(self, radio_dict: dict) -> None:
 
         self.RADIO_SCHEMA = {
             "license": {"type": str},
+            "modulation": {"type": str},
```

Comment:
Would be nice to specify which values are allowed here so folks can't set incorrect values in config.